frontend/src/metabase-types/api/settings.ts

```diff
@@ -205,6 +205,7 @@ export interface Settings {
   "ldap-configured?": boolean;
   "ldap-enabled": boolean;
   "loading-message": LoadingMessage;
+  "map-tile-server-url": string | undefined;
```

Comment:
During debugging I found this is missing, so I added it in the spirit of the boy-scout rule.